src/comparison/mod.rs

```diff
@@ -118,8 +116,17 @@ pub fn shorten(comparison: &str) -> &str {
         latest_run::NAME => latest_run::SHORT_NAME,
         none::NAME => none::SHORT_NAME,
         worst_segments::NAME => worst_segments::SHORT_NAME,
-        c => c,
-    }
+        _ => return Option::None,
+    })
+}
+
+/// Shortens a comparison name. If the name of the comparison matches one of the
+/// comparison generators, the short name of that comparison generator is
+/// returned. Otherwise the comparison name is returned without being shortened.
+/// Additional shortening logic for other comparison names may happen in the
+/// future.
```

Comment:
```suggestion
/// returned. Otherwise, the comparison name is returned without being
/// shortened. Additional shortening logic for other comparison names may happen
/// in the future.
```